src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,21 +2,27 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Login method called with login: {}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
+            logger.warn("Login failed for user: {}", login);
             throw new AuthenticationException("Username or password are incorrect");
         }
+        logger.info("User successfully logged in: {}", login);
```

Comment:
The log message for a successful login is also informative, which is excellent for monitoring user activity.